lib/eventful/helpers/notifier.ex

```diff
@@ -2,16 +2,20 @@ defmodule Eventful.Notifier do
   require Logger
   require HTTPoison
   alias Eventful.Resources
+  alias Eventful.Repo
   use Task
 
   def fanout(event) do
     Enum.each(Resources.get_subscriptions_for_topic(event.topic_id), fn(subscription) ->
-      Task.async(__MODULE__, :fire, [event, subscription])
+      max_retries = if subscription.max_retries != nil, do: subscription.max_retries, else: System.get_env("max_retries")
```

Comment:
I think you can just write it as 
subscription.max_retries || System.get_env("max_retries")